src/java/fr/paris/lutece/plugins/blog/service/BlogParameterService.java

```diff
@@ -16,5 +22,37 @@ public static BlogParameterService getInstance( )
     {
         return _singleton;
     }
-    
+
+    public void updateNumberMandatoryTags(String nbMandatoryTags)
+    {
+        int valueNbMandatoryTags = 0;
+        try
+        {
+            valueNbMandatoryTags = Integer.parseInt(nbMandatoryTags);
+        }
+        catch (NumberFormatException e)
+        {
+            AppLogService.error("Incorrect value for number mandatory tags", e);
+        }
+
+        if(valueNbMandatoryTags<0)
+        {
+            valueNbMandatoryTags=0;
+        }
+
+        DefaultUserParameterHome.update( DSKEY_DEFAULT_NUMBER_MANDATORY_TAGS, Integer.toString(valueNbMandatoryTags));
```

Comment:
What is the benefit of using DefaultUserParameterHome instead of DatastoreService ?
Is this parameter user related ?